ibis/pyspark/compiler.py

```diff
@@ -280,6 +283,188 @@ def compile_greatest(t, expr):
         return F.greatest(*src_columns)
 
 
+@compiles(ops.Least)
+def compile_least(t, expr):
+    op = expr.op()
+
+    src_columns = t.translate(op.arg)
+    if len(src_columns) == 1:
+        return src_columns[0]
+    else:
+        return F.least(*src_columns)
+
+
+@compiles(ops.Abs)
+def compile_abs(t, expr):
+    op = expr.op()
+
+    src_column = t.translate(op.arg)
+    return F.abs(src_column)
+
+
+@compiles(ops.Round)
+def compile_round(t, expr):
+    op = expr.op()
+
+    src_column = t.translate(op.arg)
+    scale = op.digits.op().value if op.digits is not None else 0
+    rounded = F.round(src_column, scale=scale)
+    if scale == 0:
+        rounded = rounded.astype('long')
```

Comment:
Why do we need to cast type here?

Reply:
Expected result is np.int64 when decimals or scale is 0. https://github.com/ibis-project/ibis/blob/master/ibis/tests/backends.py#L34